src/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const artistRouter = require('./routes/artist');
+const albumRouter = require('./routes/albums');
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/artists', artistRouter);
+app.use('/artists', albumRouter);
+
+app.use('/albums', albumRouter);
```

Comment:
You could group these together. I see why you would list them like this (and this approach would be be preferred for larger files with a much larger amount of routes) but in this case its fine to lump them together